queries.sql

```diff
@@ -22,4 +22,76 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 
 /*Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*---------------------*/
+
+BEGIN;
+
+update animals set species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+BEGIN;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+
+SELECT * FROM animals;
+
+COMMIT;
+
+/*---------------------*/
```

Comment:
- [ ] After the change was done, verify the COMMIT was done before you BEGIN.

### Sample:
```
   SELECT species from animals; -- verify the chnage while ondly calling species not all animals
   ROLLBACK;

   SELECT species from animals; 
```